src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -500,22 +500,34 @@ public override Size ClientSize
             }
         }
 
-        public override bool CursorVisible
+        public override bool CursorGrabbed
```

Comment:
This should throw on both get and set if it is not implemented, no matter the current or incoming value.

Reply:
Based on [this conversation](https://github.com/opentk/opentk/pull/589#pullrequestreview-53755951) I actually wanted to get rid of the `NotImplementedException` altogether, but apparently Frassle [didn't want it removed?](https://github.com/VPeruS/opentk/pull/3#pullrequestreview-92068072) Has this changed altogether?

Reply:
@Frassle Input on this? I don't recall this being mentioned before.

Reply:
Trying to strike a balance between being easy to use and explicit about operations that aren't supported. I'd think ideally asking if its grabbed should always return true, setting grabbed to true should be fine (noop) and setting grabbed to false should raise PlatformNotSupportedException with a message explaining that there's no way to ungrab the cursor because we don't have a window.

Reply:
At very least, the fact this throws exceptions should be exposed in xmldoc. Especially when exceptions are only thrown under certain runtime conditions, which a consumer may not be testing for.

Reply:
Agreed.

Reply:
Added